src/main/java/uk/gov/ons/census/casesvc/model/entity/Event.java

```diff
@@ -17,7 +17,8 @@
 
   @ManyToOne private UacQidLink uacQidLink;
 
-  @Column private LocalDateTime eventDate;
+  @Column(columnDefinition = "timestamp with time zone")
+  private LocalDateTime eventDate;
```

Comment:
This should be `OffsetDateTime`

Reply:
also don't need the `(columnDefinition = "timestamp with time zone")` part

Reply:
done